packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1951,10 +1945,37 @@ async function handler(
   }
 
   if (bookingRequiresPayment) {
+    // If team event get team owner's payment credential
+    let paymentCredentialUserId = organizerUser.id;
+
+    if (isTeamEventType || eventType.parentId) {
+      const teamMembers = await prisma.team.findFirst({
+        where: {
+          eventTypes: {
+            some: {
+              id: eventType.parentId || eventType.id,
+            },
+          },
+        },
+        select: {
+          members: true,
+        },
+      });
+      const teamOwner = teamMembers?.members.find((member) => member.role === "OWNER");
+
+      if (teamOwner) {
+        paymentCredentialUserId = teamOwner.userId;
```

Comment:
If the booking comes from a team event type and Stripe is enabled, then we should use the team owner's Stripe credentials. If not found then throw an error.